validator/src/validator.rs

```diff
@@ -132,7 +134,8 @@ impl<TValidatorNetwork: ValidatorNetwork> Validator<TValidatorNetwork>
 where
     PubsubId<TValidatorNetwork>: std::fmt::Debug + Unpin,
 {
-    const PRODUCER_TIMEOUT: Duration = Duration::from_millis(Policy::BLOCK_PRODUCER_TIMEOUT);
+    const MAXIMUM_PRODUCER_TIMEOUT: u64 = 30_000; // 30 seconds
```

Comment:
It can be done but it always needs to get converted to a `u64` at runtime which kinda defeats the purpose of it being a `const` since it's only being used for clamping. Not a fan of it.

Reply:
We need a `Duration` in the end to pass into `sleep()`. So you can just convert the `u64` timeout into a `Duration` and then use `Duration::clamp` to restrict the range.